org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -20,6 +20,12 @@ <h1>Change History</h1>
 
 <h2>Snapshot Build @qualified.bundle.version@ (@build.date@)</h2>
 
+<h3>New Features</h3>
+<ul>
+  <li>Synthetic classes are filtered out during generation of report
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/668">#668</a>).</li>
```

Comment:
What is our rule here: Do we refer to the PR or to the issue?

Reply:
@marchof I prefer to refer to actual changes, so to PR,  and was following this principle.